pkg/services/app/add_test.go

```diff
@@ -608,3 +607,57 @@ var _ = Describe("Add", func() {
 		})
 	})
 })
+
+var _ = Describe("sanitizeRepoUrl", func() {
+
```

Comment:
I'm not sure if this is some config on your editor, but we should avoid adding a blank line at the beginning and the end of functions.